Safewaan/HonkedIn
# Users can edit their own comments on a forum

## Trello ticket link

[As a user, I can edit my comments on a forum.](https://trello.com/c/6cvQCGZh/39-as-a-user-i-can-edit-my-comments-on-a-forum)


## Implementation description
* Users can click into a forum and edit their own comments
* Users can choose not to continue editing their comment
* If a user edits their comment, the timestamp will be updated as well 



## Steps to test
1. Please go to http://localhost:3000/forums
2. Please select a forum where you have already commented or create a comment
3. Edit your comment
4. Wait for the page to automatically refresh and ensure that a toast message appears and your comment is edited 
5. Please ensure that the timestamp for the comment has been updated as well 


## What should reviewers focus on?
* You should only be able to edit your own comments 
* You should be able to edit a comment as many times as you would like to
* The timestamp updates whenever a comment is updated
